docs/graphql/core/migrations/manage-metadata.rst

```diff
@@ -157,6 +157,41 @@ when a new column has been added to a table via an external tool such as ``psql`
    Reloading may result in inconsistent metadata status. You may need to resolve
    all inconsistent objects manually or delete them. After that, you need to reload
    metadata again.
+   
+   
+ .. _reset_metadata_manual:
```

Comment:
There is something wrong with the indentation here. It messes up the note in the previous section:

![image](https://user-images.githubusercontent.com/5722022/95549571-d3d60700-0a07-11eb-9e69-84bf6be1c0d0.png)